pkg/svcat/service-catalog/plan.go

```diff
@@ -155,3 +185,42 @@ func (sdk *SDK) RetrievePlanByClassAndPlanNames(className, planName string,
 	}
 	return &searchResults.Items[0], nil
 }
+
+// CreatePlan creates a new plan.
+func (sdk *SDK) CreatePlan(opts CreatePlanFromOptions,
```

Comment:
nit: I suggest changing the name of the opts to `CreatePlanOptions` to match the function name where it is used.

`nit` is a thing that we do for items that don't _have_ to be changed, but as long as you are making other changes, and agree with the suggestion, it would be nice to change as well at the same time.